README.md

```diff
@@ -1,3 +1,58 @@
-Voron Tap
+# Voron Tap
+
+![Voron-Tap3](https://user-images.githubusercontent.com/219295/206089801-1c3b0685-ac0b-460a-9027-2fd87e1cb79e.gif)
```

Comment:
Can you add this GIF to the repository and reference it here, instead of using githubusercontent?

Reply:
Yes, if you prefer. However, I followed github's recommended approach here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

Reply:
That is talking about images in issues and pull requests, not images in the repository readme.  Images in the repository readme should ideally be relative links to images in the same repository, as documented here

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

This makes it easier for clones or branches to modify the image for example, or to get a local copy of the image for viewing offline.

Reply:
done